forge-gui/res/cardsfolder/upcoming/the_fall_of_kroog.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Fall of Kroog
+ManaCost:4 R R
+Types:Sorcery
+A:SP$ ChoosePlayer | ValidTgts$ Player.Opponent | TgtPrompt$ Select target opponent | Choices$ TargetedPlayer | SubAbility$ DBDestroy | StackDescription$ SpellDescription | RememberOriginalTargets$ True | SpellDescription$ Choose target opponent. Destroy target land that player controls. CARDNAME deals 3 damage to that player and 1 damage to each creature they control.
```

Comment:
I think this would be better as a Pump, just cleaner
also `ValidTgts$ Opponent` and get rid of TgtPrompt